external/twitter/src/main/scala/org/apache/spark/streaming/twitter/TwitterUtils.scala

```diff
@@ -40,7 +40,7 @@ object TwitterUtils {
       twitterAuth: Option[Authorization],
       filters: Seq[String] = Nil,
       storageLevel: StorageLevel = StorageLevel.MEMORY_AND_DISK_SER_2
-    ): ReceiverInputDStream[Status] = {
+    ): ReceiverInputDStream[Status] = ssc.withScope {
```

Comment:
Wait, why at all have customScopeName in DStream, when you can always set the custom scope name using `withScope(someScopeName)`?